### PR TITLE
add mpp query count metrics

### PR DESCRIPTION
**Pull request by windtalker**
Signed-off-by: xufei <xufeixw@mail.ustc.edu.cn>

### What problem does this PR solve?

Issue Number: ref #5095

Problem Summary:

### What is changed and how it works?
Add mpp query count metrics
### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by SeaRise — APPROVED** [review]
LGTM

**Review by ywqzzy — APPROVED** [review]
LGTM

File: dbms/src/Flash/Mpp/MPPTaskManager.cpp [hunk 1]
```diff
@@ -153,6 +154,7 @@ void MPPTaskManager::abortMPPQuery(UInt64 query_id, const String & reason, Abort
         /// just to double check the query still exists
         if (it != mpp_query_map.end())
             mpp_query_map.erase(it);
```
Comment (SeaRise) [inline dbms/src/Flash/Mpp/MPPTaskManager.cpp:156]: or
```
if (it != mpp_query_map.end())
{
    mpp_query_map.erase(it);
    GET_METRIC(tiflash_mpp_task_manager, type_mpp_query_count).Set(mpp_query_map.size());
}         
```
?

Comment (windtalker) [reply dbms/src/Flash/Mpp/MPPTaskManager.cpp:156]: It's ok since it will be removed in the next pr.

**Review by yibin87 — APPROVED** [review]
LGTM

**Pull request merged**